examples/runners/wgpu/src/graphics.rs

```diff
@@ -115,7 +115,7 @@ async fn run(
         // the resources are properly cleaned up.
         let _ = (&instance, &adapter, &module, &pipeline_layout);
 
-        *control_flow = ControlFlow::Wait;
+        *control_flow = ControlFlow::Poll;
```

Comment:
This got changed to Wait on the android PR, why are we changing it back?

Reply:
If you put it to wait the window only gets updated when you move the mouse or other events get triggered.